demos/ripple.html

```diff
@@ -39,17 +39,17 @@
     <script src="assets/material-design-lite.js" charset="utf-8"></script>
     <script>
       (function(global) {
-        'use strict';
-        let surfaces = document.querySelectorAll('.surface');
-        for (let i = 0; i < surfaces.length; i++) {
-          let surface = surfaces[i];
-          let component = global.mdl.Ripple.attachTo(surface);
-          let button = surface.parentNode.querySelector('button');
-          button.addEventListener('click', () => {
-            let rect = surface.getBoundingClientRect();
+        var surfaces = document.querySelectorAll('.surface');
+        [].forEach.call(surfaces, function(surface) {
+          var rippleDom = global.mdl.Ripple.buildDom();
+          var component = global.mdl.Ripple.attachTo(surface, rippleDom);
+          var button = surface.parentNode.querySelector('button');
+          button.addEventListener('click', function() {
+            var rect = surface.getBoundingClientRect();
```

Comment:
@Garbee This is a script tag embedded in a demo HTML page so ES2015 preprocessing doesn't work.